<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArbitAssignments</class>
 <widget class="QWidget" name="ArbitAssignments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>_(&quot;Assignments&quot;)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>_(&quot;Master move:&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClickableLabel" name="labelMasterMove">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSelectMaster">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>_(&quot;Select&quot;)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxHideAssigned">
        <property name="text">
         <string>_(&quot;Hide players with an assigned move&quot;)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="treeViewPlayers">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="buttonAssign">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>_(&quot;Assign move&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSuppressMove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>_(&quot;Indicate that the selected player(s) did not play&quot;)</string>
          </property>
          <property name="text">
           <string>_(&quot;Suppress move&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonEndTurn">
          <property name="toolTip">
           <string>_(&quot;Validate the current turn and start a new one&quot;)</string>
          </property>
          <property name="text">
           <string>_(&quot;&amp;End turn&quot;)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">misc_helpers.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
